zio/core/src/main/scala/tofu/zioInstances/ZioTofuInstance.scala

```diff
@@ -78,10 +79,12 @@ object ZioTofuInstance {
   }
 }
 
-class ZioTofuErrorsToInstance[R, E, E1] extends ErrorsTo[ZIO[R, E, *], ZIO[R, E1, *], E] {
+class ZioTofuErrorsToInstance[R, E, E1](implicit lens: Extract[E1, E])
```

Comment:
So now the only use of the `lens` parameter is to `Extract` something from `Nothing`
This is perfectly legit and could not touch performance (since mapError is never applied).
But it looks like we need at least one additional instance , e.g.
```scala
final def zioTofuExtractErrorsInstance[R, E, E1: * Extract E1]: ZioTofuErrorsToInstance[R, E, E1] = 
  new ZioTofuErrorsToInstance
```

To justify this encoding.